data/campaigns/tutorial/scenarios/03_Tutorial_part_3.cfg

```diff
@@ -0,0 +1,803 @@
+            facing=sw
+        [/modify_unit]
+        [move_unit]
+            id=DumboBait
+            to_x,to_y=7,4
+        [/move_unit]
+
+        [item]
+            image=misc/highlight-hex.png
+            [filter_adjacent_location]
+                [filter]
+                    id=Elf1,Elf2
+                [/filter]
+            [/filter_adjacent_location]
+        [/item]
+
+        # po: "encircle" is an obscure word, but one of the tips in the "wesnoth" text domain uses "encircling", and the tutorial shows a diagram with highlighted hexes while showing this text
+        {TALK_ABOUT Dumbo ( _ "On the next turn the other two elves could encircle our unit. When two units are on opposite sides of an enemy, their overlapping <i>Zones of Control</i> mean the unit is trapped - it can only move one hex before being stopped by the ZoC. Meanwhile the injured elf could heal in a village.")}
+
+        [move_unit]
+            id=DumboBait
+            to_x,to_y=9,8
+        [/move_unit]
+        [remove_item]
+            image=misc/highlight-hex.png
+        [/remove_item]
+        [move_unit]
+            id=Elf1
+            to_x,to_y=6,6
+        [/move_unit]
+        [move_unit]
+            id=Elf2
+            to_x,to_y=11,5
+        [/move_unit]
+        [move_unit]
+            id=Dumbo
+            to_x,to_y=9,13
+        [/move_unit]
+        [unhide_unit]
+            id=DumboBait
+        [/unhide_unit]
+
+        {VARIABLE spoke_about_dumbo_trap yes}
+    [/event]
+
+    # Explaining the Assassin
+    [event]
+        name=recruit,recall
+        [filter]
+            side=1
+            type_adv_tree=Orcish Assassin
+        [/filter]
+
+        [message]
+            speaker=narrator
+            image=$unit.profile
+            caption= _ "Orcish Assassin"
+            message= _ "The Assassin’s ranged attack has <i>poison</i>, an enemy that’s been hit by it will take 8 hitpoints of damage each turn until they either drop to 1 hp or heal in a village. The shaman’s healing will prevent this damage, but won’t cure it (and the unit won’t get the 4 hp healing from the shaman either). That attack also has the <i>marksman</i> special, it always has at least a 60% chance of hitting, even when targeting an elf hiding in forest."
+        [/message]
+
+        [message]
+            speaker=narrator
+            image=$unit.profile
+            caption= _ "Orcish Assassin"
+            message= _ "Assassins have 60% defense on flat, and 70% in forests and villages, however they have negative <i>resistances</i>, meaning they take more damage from blades and piercing weapons. The resistances are shown when you hover over the unit’s hitpoints."
+        [/message]
+    [/event]
+
+    # Explaining the Archer
+    [event]
+        name=recruit,recall
+        [filter]
+            side=1
+            type_adv_tree=Orcish Archer
+        [/filter]
+
+        [message]
+            speaker=narrator
+            image=$unit.profile
```

Comment:
$unit.profile? Shouldn't this line be spoken by the lvl3 Orcish advisor? It's weird for the recruited unit to speak of "The Orcish Archer" in the third person.

Reply:
This matches S2.